<?xml version="1.0"?>
<launch>
   <node name="sirona_node" pkg="sirona" type="sirona_node" output="screen">
      <param name="sub_map_topic"              type="string" value="map"                       />
      <param name="sub_target_topic"           type="string" value="/move_base_simple/goal"    />
      <param name="sub_add_obstacle_topic"     type="string" value="add_obstacle"              />
      <param name="sub_rm_obstacle_topic"      type="string" value="rm_obstacle"               />
      <param name="pub_state_topic"            type="string" value="navigation/state"          />
      <param name="pub_path_topic"             type="string" value="path"                      />
      <param name="pub_grid_topic"             type="string" value="navigation/map"            />
      <param name="pub_grid_ob_topic"          type="string" value="navigation/obstacles"      />
      <param name="pub_move_pause_topic"       type="string" value="path_control/pause"        />
      <param name="map_frame"                  type="string" value="map"                       />
      <param name="robot_frame"                type="string" value="base_footprint"            />
      <param name="robot_radius"               type="double" value="0.35"                      />
      <param name="dt_radius"                  type="double" value="0.2 "                      />
      <param name="free_robot_pos_factor"      type="double" value="0.5 "                      />
      <param name="obstacle_react_dist"        type="double" value="1.5 "                      />
      <param name="obstacle_block_time"        type="double" value="2   "                      />
   </node>
</launch>